.github/workflows/test.yaml

```diff
@@ -0,0 +1,80 @@
+name: build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  IP: 127.0.0.1
+
+jobs:
+  lint:
+    strategy:
+      matrix:
+        go-version: [ 1.14.x ]
```

Comment:
We use 1.13 everywhere. Any reason for the change?

Reply:
In the previous Travis CI setup, go 1.14.x was used, so I didn't change it, I will update with go 1.13, ok.

Reply:
done.